docs/src/main/asciidoc/qute-reference.adoc

```diff
@@ -1176,12 +1176,34 @@ engineBuilder.addValueResolver(ValueResolver.builder()
 [[template-locator]]
 ==== Template Locator
 
-Manual registration is sometimes handy but it's also possible to register a template locator using `EngineBuilder.addLocator()`.
+Manual registration is sometimes handy but it's also possible to register a template locator.
```

Comment:
I know that you kept the original wording but it sounds a bit strange... maybe we should use something like _"A template can be either registered manually or automatically via a template locator. The locators are used whenever the `Engine.getTemplate()` method is called and the engine has no template for a given id stored in the cache.
The locator is responsible to use the correct character encoding when reading the contents of a template."_

And then move all the quarkus-specific stuff under the [Quarkus Integration](https://quarkus.io/guides/qute-reference#quarkus_integration) section and reference this new section in the NOTE, i.e. "NOTE: In Quarkus, all templates from the `src/main/resources/templates` are located automatically and the encoding set via `quarkus.qute.default-charset` (UTF-8 by default) is used. Custom locators can be registered via the `@Locate` annotation.".

WDYT?

Reply:
Looks much better this way, I added "template locator registration" to Quarkus Integration section, please let me know if it needs correction as well.